### PR TITLE
Promoting a few logs to Info

### PR DESCRIPTION
**Pull request by boks1971**
Also, adding a couple of more info logs which I will remove later
after some debugging.

### CODE REVIEW

File: pkg/rtc/wrappedreceiver.go [hunk 1]
```diff
@@ -166,24 +167,26 @@ func (d *DummyReceiver) SendPLI(layer int32, force bool) {
...
+	defer d.settingsLock.Unlock()
 	if r, ok := d.receiver.Load().(sfu.TrackReceiver); ok {
+		d.pausedValid = false
 		r.SetUpTrackPaused(paused)
 	} else {
-		d.settingsLock.Lock()
 		d.pausedValid = true
 		d.paused = paused
-		d.settingsLock.Unlock()
 	}
```
Comment (boks1971) [inline pkg/rtc/wrappedreceiver.go:178]: Making these changes to protect these values racing during upgrade of receiver. Upgrade swaps receiver and then proceeds to transfer from dummy receiver -> real receiver. So, it was possible for those to race.

File: pkg/sfu/downtrack.go [hunk 2]
```diff
@@ -694,13 +695,13 @@ func (d *DownTrack) CloseWithFlush(flush bool) {
...
 func (d *DownTrack) SetMaxSpatialLayer(spatialLayer int32) {
+	d.logger.Infow("setting max spatial layer", "layer", spatialLayer)
 	changed, maxLayers, currentLayers := d.forwarder.SetMaxSpatialLayer(spatialLayer)
 	if !changed {
 		return
 	}
 
-	if d.onMaxLayerChanged != nil && d.kind == webrtc.RTPCodecTypeVideo &&
-		maxLayers.SpatialGreaterThanOrEqual(currentLayers) && d.bound.Load() {
+	if d.onMaxLayerChanged != nil && d.kind == webrtc.RTPCodecTypeVideo && maxLayers.SpatialGreaterThanOrEqual(currentLayers) {
```
Comment (boks1971) [inline pkg/sfu/downtrack.go:704]: I think this is where the issue is with stress test failure. Subscription update happening before bind. We had an update in Bind callback, but that was causing a different race and hence removed.

File: pkg/rtc/dynacastmanager.go [hunk 3]
```diff
@@ -174,6 +174,12 @@ func (d *DynacastManager) updateMaxQualityForMime(mime string, maxQuality liveki
 func (d *DynacastManager) update(force bool) {
 	d.lock.Lock()
 
+	d.params.Logger.Infow("processing quality change",
+		"force", force,
+		"committedMaxSubscribedQuality", d.committedMaxSubscribedQuality,
+		"maxSubscribedQuality", d.maxSubscribedQuality,
+	)
+
```
Comment (boks1971) [inline pkg/rtc/dynacastmanager.go:182]: Will remove this after debugging

File: pkg/rtc/dynacastquality.go [hunk 4]
```diff
@@ -58,6 +58,7 @@ func (d *DynacastQuality) OnSubscribedMaxQualityChange(f func(maxSubscribedQuali
 }
 
 func (d *DynacastQuality) NotifySubscriberMaxQuality(subscriberID livekit.ParticipantID, quality livekit.VideoQuality) {
+	d.params.Logger.Infow("setting subscriber max quality", "mime", d.params.MimeType, "subscriberID", subscriberID, "quality", quality.String())
```
Comment (boks1971) [inline pkg/rtc/dynacastquality.go:61]: probably downgrade these to `Debugw` after debugging (the one below as well)

File: pkg/rtc/signalhandler.go [hunk 5]
```diff
@@ -56,6 +56,10 @@ func HandleParticipantSignal(room types.Room, participant types.LocalParticipant
 			pLogger.Warnw("could not update subscription", err,
 				"tracks", msg.Subscription.TrackSids,
 				"subscribe", msg.Subscription.Subscribe)
+		} else {
+			pLogger.Infow("updated subscription",
+				"tracks", msg.Subscription.TrackSids,
+				"subscribe", msg.Subscription.Subscribe)
```
Comment (boks1971) [inline pkg/rtc/signalhandler.go:62]: Going to keep this as it is useful to see when subscriptions were enabled/disabled.

File: pkg/rtc/uptrackmanager.go [hunk 6]
```diff
@@ -170,7 +170,7 @@ func (u *UpTrackManager) SetPublishedTrackMuted(trackID livekit.TrackID, muted b
 		track.SetMuted(muted)
 
 		if currentMuted != track.IsMuted() {
-			u.params.Logger.Debugw("mute status changed", "trackID", trackID, "muted", track.IsMuted())
+			u.params.Logger.Infow("mute status changed", "trackID", trackID, "muted", track.IsMuted())
```
Comment (boks1971) [inline pkg/rtc/uptrackmanager.go:173]: going to keep this at info

File: pkg/rtc/uptrackmanager.go [hunk 7]
```diff
@@ -232,7 +232,7 @@ func (u *UpTrackManager) UpdateSubscriptionPermission(
 	// store as is for use when migrating
 	u.subscriptionPermission = subscriptionPermission
 	if subscriptionPermission == nil {
-		u.params.Logger.Debugw(
+		u.params.Logger.Infow(
```
Comment (boks1971) [inline pkg/rtc/uptrackmanager.go:235]: will revert this

File: pkg/rtc/uptrackmanager.go [hunk 8]
```diff
@@ -241,7 +241,7 @@ func (u *UpTrackManager) UpdateSubscriptionPermission(
 		return nil
 	}
 
-	u.params.Logger.Debugw(
+	u.params.Logger.Infow(
```
Comment (boks1971) [inline pkg/rtc/uptrackmanager.go:244]: will revert this

**Review by davidzhao — APPROVED** [review]

**Pull request merged**